### DISCUSSION TITLE
Field Membership Profileは取得(表示)できますか？

### DISCUSSION THREAD

**Discussion by phys-ken** [kosugitti/exametrika#11] [Q&A] [answered]
自分の手元の試験データでBiclustringが実行できて感動しています。

test data engineeringの292ページにある、Field Membership Profileをこのパッケージで表示することはできますか？
Field analysisを行いたいです。教えていただけると大変助かります。

**Comment by kosugitti** [comment, marked as answer]
可能です。
関数が戻してくるオブジェクトの中にFieldMembership, ClassMembershipが入ってます。
`ret$FieldMembership`のようにして呼び出してください。

**Reply by phys-ken** [reply]
ありがとうございました！できました！感動しています。
